sample.env

```diff
@@ -4,6 +4,9 @@
 # REACT_APP_UNCHAINED_BITCOIN_HTTP_URL=http://api.bitcoin.localhost
 # REACT_APP_UNCHAINED_BITCOIN_WS_URL=ws://api.bitcoin.localhost
 # connect to dev cluster
+
+REACT_APP_LOG_LEVEL=info
```

Comment:
thinking ahead - i assume we want `debug` for dev and prod environments?

Reply:
ya - the question is what do we want? i'm going to default to debug as i'd rather have more information for user facing issues than not, unless anyone else has objections?